app/(pages)/sponsors/page.tsx

```diff
@@ -1,15 +1,141 @@
+        <SponsorSection title='Advanced Sponsors'/>
+        <Stack direction="column" spacing={8} justifyContent="center" alignItems="center">
+          <Stack 
+            direction={{ xs: 'column', md: 'row', lg: 'row'}} 
+            spacing={10}
+            justifyContent="center"
+            alignItems="center"
+            sx= {{ width: { xs: '80%', sm: '65%', md: '85%', lg: '70%'} }}
+          >
+            <SponsorRows maxWidth='320px' imageComponent='/sponsors/cochlear.jpg' imageName='Cochlear'></SponsorRows>
+            <SponsorRows maxWidth='340px' imageComponent='/sponsors/aecom.jpg' imageName='Aecom'></SponsorRows>
+            <SponsorRows maxWidth='340px' imageComponent='/sponsors/dolby.jpg' imageName='Dolby'></SponsorRows>
+
+          </Stack>
+          <Stack 
+            direction={{ xs: 'column', md: 'row', lg: 'row'}} 
+            spacing={10}
+            justifyContent="center"
+            alignItems="center"
+            sx= {{ width: { xs: '80%', sm: '65%', md: '85%', lg: '70%'} }}
+          >
+            <SponsorRows maxWidth='400px' imageComponent='/sponsors/sportsGrid.jpg' imageName='Sports Grid'></SponsorRows>
+            <SponsorRows maxWidth='450px' imageComponent='/sponsors/systa.jpg' imageName='Systra'></SponsorRows>
+          </Stack>
+          <Stack 
+            direction={{ xs: 'column', md: 'row', lg: 'row'}} 
+            spacing={10}
+            justifyContent="center"
+            alignItems="center"
+            sx= {{ width: { xs: '80%', sm: '65%', md: '85%', lg: '70%'} }}
+          >
+            <SponsorRows maxWidth='300px' imageComponent='/sponsors/appian.svg' imageName='Appian'></SponsorRows>
+            <SponsorRows maxWidth='400px' imageComponent='/sponsors/projectServices.jpg' imageName='Project Services'></SponsorRows>
+            <SponsorRows maxWidth='450px' imageComponent='/sponsors/stantec.jpg' imageName='Stantec'></SponsorRows>
+          </Stack>
+        </Stack>
+      </Box>
+      <Box width='95%' mx='auto' mb={15}>
+        <SponsorSection title='Foundation Sponsors'/>
+        <Stack direction="column" spacing={8} justifyContent="center" alignItems="center">
+          <Stack 
+            direction={{ xs: 'column', md: 'row', lg: 'row'}} 
+            spacing={10}
+            justifyContent="center"
+            alignItems="center"
+            sx= {{ width: { xs: '80%', sm: '65%', md: '85%', lg: '70%'} }}
+          >
+            <SponsorRows maxWidth='450px' imageComponent='/sponsors/honeywell.jpg' imageName='Honeywell'></SponsorRows>
+            <SponsorRows maxWidth='400px' imageComponent='/sponsors/flowTraders.jpg' imageName='Flow Traders'></SponsorRows>
+          </Stack>
+          <Stack
+            direction={{ xs: 'column', md: 'row', lg: 'row'}} 
+            spacing={10}
+            justifyContent="center"
+            alignItems="center"
+            sx= {{ width: { xs: '80%', sm: '65%', md: '85%', lg: '70%'} }}
```

Comment:
Alot of code repetition for this Stack and its styling, perhaps make it a component